src/utils/events/move-input.js

```diff
@@ -24,9 +26,29 @@ export default class MoveInput {
   }
 
   handler(event) {
```

Comment:
This is the only place I am aware of where we use a noun to `name` a method. Usually we use `verbNoun` or `verb`.